### PR TITLE
Make AccountHandle::get_alias_output() public

### LINKED ISSUE #214: Make `AccountHandle::get_alias_output()` accessible
## Description

Make `AccountHandle::get_alias_output()` accessible also outside the crate

## Motivation

To use an already existing fuction for retrieving alias data.

## Are you planning to do it yourself in a pull request?

Yes

### PR DESCRIPTION
**Pull request by lucagiorgino**
# Description of change

modified `AccountHandle::get_alias_output()` visibility from `pub(crate)` to `pub`

## Links to any relevant issues

Closes #214 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: sdk/src/wallet/account/operations/transaction/high_level/create_alias.rs [hunk 1]
```diff
@@ -154,7 +154,7 @@ impl AccountHandle {
     }
 
     /// Get an existing alias output
```
Comment (Thoralf-M) [inline sdk/src/wallet/account/operations/transaction/high_level/create_alias.rs]:
```suggestion
    /// Get an unspent alias output
```

**Comment by thibault-martinez** [comment]
This doesn't really look like an API that should be exposed tbh... Maybe we should add another one that just returns an Option output. Or do you actually need that behaviour ?

**Comment by lucagiorgino** [comment]
I could use client functions and retrieve Alias information from a node, the idea was to reuse code

**Comment by thibault-martinez** [comment]
Hi @lucagiorgino, thanks for the PR but we discussed internally and we don't really like this API to be public. We will add some more features to allow you to do that in a better way as described in https://github.com/iotaledger/iota-sdk/issues/341. We'll do that as part of our next sprint starting on Monday.

**Pull request closed without merging**